.github/workflows/phpunit.yml

```diff
@@ -1,14 +1,9 @@
 name: PHPUnit tests
 
-on: pull_request
-
-### TODO whitelist branches:
- # branches:
-#   only:
-#     - master
-#     - master-stable
-#     - /^branch-.*$/
-#     - /^feature\/.*$/
+on:
+  pull_request:
+  push:
+    branches: [master, master-stable, 'branch-*']
```

Comment:
Why does this one list three branch patterns, while the other two (particularly js-tests.yml) only specifies "master"?

Reply:
It was an overlook from my side. Add it to `jstests` actions aswell